windtunnel/windtunnel.py

```diff
@@ -47,21 +47,73 @@
 class WindTunnel:
     """WindTunnel scenario."""
 
-    def __init__(self, inputs_dir: str = "./inductiva_input/"):
-        """Initializes the `WindTunnel` scenario object.
+    def __init__(self, dimensions: tuple[int, int, int] = (20, 10, 8)):
+        """
+        Initializes the wind tunnel with given dimensions.
+
+        Parameters:
+        - dimensions: A tuple (length, width, height) for the tunnel size.
         """
         self._walls = {
-            "x_min": -6,
-            "x_max": 14,
-            "y_min": -5,
-            "y_max": 5,
+            "x_min": -dimensions[0] * 0.3,
```

Comment:
consider leaving a small comment, on why choosing these constants (e.g it is centered on y, but is not quite centered on x so that when we place the object at the origin it will appear closer to the "entrance" of the wind tunnel) ... something like that (in perhaps better wording).